packages/rangelink-vscode-extension/src/__integration-tests__/suite/linkGeneration.test.ts

```
@@ -27,93 +27,67 @@ standardSuite('Link Generation', (ss) => {
-      assert.strictEqual(verdict, 'pass', `Human reported FAIL: ${label} RangeLink did not navigate correctly`);
-      ss.log(`✓ wrapped-link-navigation-${tcId} — ${label} RangeLink navigated (human verified)`);
-    });
-  }
+  const runWrappedNavigationTest = async (
+    tcId: string,
+    label: string,
+    wrapperDesc: string,
+    open: string,
+    close: string,
+    suffix: string,
+  ): Promise<void> => {
```

ESLint

[error] Replace `⏎····tcId:·string,⏎····label:·string,⏎····wrapperDesc:·string,⏎····open:·string,⏎····close:·string,⏎····suffix:·string,⏎··` with `tcId:·string,·label:·string,·wrapperDesc:·string,·open:·string,·close:·string,·suffix:·string`

(prettier/prettier)